vault/external_tests/raft/raft_test.go

```diff
@@ -68,6 +70,61 @@ func raftCluster(t testing.TB, ropts *RaftClusterOpts) *vault.TestCluster {
 	return cluster
 }
 
+func TestRaft_BoltDBMetrics(t *testing.T) {
+	sink := metrics.NewInmemSink(1000000*time.Hour, 2000000*time.Hour)
+	conf := vault.CoreConfig{
+		MetricSink:    metricsutil.NewClusterMetricSink("test-cluster", sink),
+		MetricsHelper: metricsutil.NewMetricsHelper(sink, false),
+	}
+	opts := vault.TestClusterOptions{HandlerFunc: vaulthttp.Handler}
+	teststorage.RaftBackendSetup(&conf, &opts)
+	cluster := vault.NewTestCluster(t, &conf, &opts)
```

Comment:
Since you're not making use of the standbys, might want to set NumCores=1, so we don't have to wait for the nodes to be joined to the cluster.

Reply:
Ah, I didn't know about that, thanks!